docs/content/en/docs-dev/feature-status/_index.md

```diff
@@ -54,7 +54,7 @@ Please note that the phases (Incubating, Alpha, Beta, and Stable) are applied to
 | Deployment with a defined pipeline (e.g. canary, analysis) | Beta |
 | [Automated rollback](/docs/user-guide/rolling-back-a-deployment/) | Beta |
 | [Automated configuration drift detection](/docs/user-guide/configuration-drift-detection/) | Alpha |
-| [Application live state](/docs/user-guide/application-live-state/) | Incubating |
+| [Application live state](/docs/user-guide/application-live-state/) | Alpha |
```

Comment:
nice 👍  I just wonder the next release would be `v0.26.1` or `v0.27.0` cause if it's `v0.26.1`, we may need to update the docs under `/docs-v0.26.x` either 🤔

Reply:
Absolutely, let me talk about it tomorrow!

Reply:
I think the next one is definitely going to be `v0.27.0` because of this https://github.com/pipe-cd/pipecd/pull/3300.
So it is fine to update only in this place.